frontend/src/container/GridCardLayout/styles.ts

```diff
@@ -33,7 +33,14 @@ export const Card = styled(CardComponent)<CardProps>`
 	}
 
 	.ant-card-body {
-		height: calc(100% - 30px);
+		${({ $panelType }): StyledCSS =>
+			$panelType === PANEL_TYPES.TABLE
```

Comment:
the 30px is for legends in other panels ?

Reply:
+1

Reply:
Yes, this was the existing space given for legends of other types